USERS.md

```diff
@@ -679,3 +679,8 @@ Users (Alphabetically)
       D: Zero Hash is using Cilium as CNI for networking, security and monitoring features for Kubernetes clusters 
       U: CNI/ENI Networking, Network policies, Hubble
       Q: @eugenestarchenko
+
+    * N: Molnett
```

Comment:
Thanks for this, but could we move this to another PR as the review process here is different. Thanks

Reply:
Got it 👍🏻